plugins/modules/kubevirt_vm.py

```diff
@@ -324,6 +348,24 @@ def create_vm(params: Dict) -> Dict:
     return vm
 
 
+def set_wait_condition(module: AnsibleK8SModule) -> None:
+    """
+    set_wait_condition sets the wait_condition to allow waiting for the ready
+    state of the VirtualMachine depending on the module parameters running
+    and run_strategy.
+    """
+    running = module.params["running"]
+    run_strategy = module.params["run_strategy"]
+    if (
```

Comment:
supernit - you could just default to `WAIT_CONDITION_READY` and drop this conditional, adding another simple one for `run_strategy == "Manual"`.

Reply:
Good suggestion, updated!